### ISSUE TITLE
Exchanged SUPABASE variable

### ISSUE THREAD

**Issue by SnoopyDevelops** [e2b-dev/e2b#22]
`SUPABASE_SERVICE_ROLE_KEY` and `NEXT_PUBLIC_SUPABASE_ANON_KEY`  from the local Supabase is exchanged version of keys in `env.example` file.

![image](https://user-images.githubusercontent.com/54778084/232662841-4168b51f-16ce-4ea4-a8b6-1b2c3be80452.png)

![image](https://user-images.githubusercontent.com/54778084/232663053-8907d11d-b850-44fa-8cf5-d1241aa0f856.png)

Should I exchange keys in env file or keep it as in `env.example` file?

**Comment by mlejva** [comment]
Hey, the Supabase keys should be name as is specified in the `.env.example` but you should put them in a new file called `.env`

So the `.env` file will look like this:
```
NEXT_PUBLIC_SUPABASE_URL=http://localhost:54321
SUPABASE_POSTGRES_URL=postgresql://postgres:postgres@localhost:54322/postgres
NEXT_PUBLIC_API_URL=http://localhost:49155

SUPABASE_SERVICE_ROLE_KEY=<this is the value that Supabase shows for the service_role_key>
NEXT_PUBLIC_SUPABASE_ANON_KEY=<this is value that Supabase shows for the anon_key>
```

Did I understand the question correctly? Let us know if this solved your problem.

**Comment by ValentaTomas** [comment]
Thanks for catching this - the anon and role keys were accidentally switched.
It is fixed now on `main` (e1947dbc2679524e05959a3f2473ce095fc5648a).

If you are using local Supabase (`npm start`/`npm run db:start`) you don't have to replace the keys from `.env.example` because the keys you get when starting local Supabase never change.
